src/lib/layer-manager.js

```diff
@@ -173,6 +170,26 @@ export default class LayerManager {
     });
   }
 
+  pickLayerByBoundingBox({x, y, width, height}) {
```

Comment:
Yeah, I knew that. pickLayer() and pickLayerByBoundingBox() share a lot of code, so do pickLayers() and pickLayersByBoundingBox() in draw-and-pick.js.

I'm wondering if mouse event picking can also be implemented via pickLayerByBoundingBox() with width and height both set to 1. We'll need to handle picking "mode" better.